src-cpp/app/modules/main/wallet/accounts/model.h

```diff
@@ -34,10 +34,11 @@ class Model : public QAbstractListModel
     QHash<int, QByteArray> roleNames() const override;
     int rowCount(const QModelIndex&) const override;
     QVariant data(const QModelIndex& index, int role) const override;
-    void setItems(QVector<Item>& items);
+    void setItems(QVector<Item *>& items);
```

Comment:
if you define `QVector<Item *>& items` that means you will change that parameter in method. If you use `const` reference you avoid unnecessary copy.

Reply:
actually const won't avoid copying but its a better practice. have made it const now :)